Build/Common.Build.Default.props

```diff
@@ -43,7 +43,6 @@
   <!-- Change configuration name to output to a different folder -->
   <!-- POGO instrumentation -->
   <PropertyGroup>
-    <IsPogoBuild />
```

Comment:
Honestly, I don't recall why I put it in the first place. I probably copied it from somewhere else.
It might be fine to remove though since we haven't had any problem with non-pogo builds lately